tests/func/test_import_url.py

```diff
@@ -42,6 +43,10 @@ def test(self):
 
 
 class TestShouldRemoveOutsBeforeImport(TestDvc):
```

Comment:
This can also be refactored.
prepare external resource using tmp_path_factory fixture (eg `tmp_path_factory.mktemp("external_source") / "file"`

Reply:
I copied this before refactoring, then forgot to remove the original :)